### PR TITLE
feat: change restore and backup wallet inputs

### PR DESCRIPTION
**Pull request by teebszet**
# 🔘 PR Type
- [x] Enhancement

# 📜 Background
Issue Link: https://linear.app/xverseapp/issue/ENG-2604/patch-demonic-vulnerability
Context Link (if applicable):

# 🔄 Changes
- restore wallet screen inputs are 12 or 24 input boxes
- create wallet flow will verify based on guessing 3 correct words
- disable user select and copy on any component displaying the whole seed phrase
- add phosphor icon set according to zeroheight https://zeroheight.com/0683c9fa7/p/66541d-icons/b/095f0e
- fix lint errors on routes/index.ts

# 🖼 Screenshot / 📹 Video

https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/7ed04fab-cedb-4741-9df0-3bd8cc8e360f

https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/d83fd456-9360-4f9c-ac5b-24a20ca65445

https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/d2b4a977-284f-4e42-b98d-cecc700130db

# ✅ Review checklist
Please ensure the following are true before merging:

- [x] Code Style is consistent with the project guidelines.
- [x] Code is readable and well-commented.
- [x] No unnecessary or debugging code has been added.
- [x] Security considerations have been taken into account.
- [x] The change has been manually tested and works as expected.
- [x] Breaking changes and their impacts have been considered and documented.
- [x] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: src/app/screens/backupWalletSteps/verifySeed.tsx [hunk 1]
```diff
@@ -32,54 +26,125 @@ const TransparentButtonContainer = styled.div((props) => ({
...
+  const [correctCounter, setCorrectCounter] = useState(0);
+  const [quiz, setQuiz] = useState<{ words: string[]; answer: string; nth: string }>({
+    words: [],
+    answer: '',
+    nth: '',
+  });
 
-  const cleanMnemonic = (rawSeed: string): string =>
-    rawSeed.replace(/\s\s+/g, ' ').replace(/\n/g, ' ').trim();
+  const generateWords = useCallback(() => {
```
Comment (yknl) [inline src/app/screens/backupWalletSteps/verifySeed.tsx:103]: Would this have a chance to generate a duplicate correct word which would show as incorrect if clicked on.

Comment (teebszet) [reply src/app/screens/backupWalletSteps/verifySeed.tsx:103]: Yeah that's a good one. Will handle that case

Comment (teebszet) [reply src/app/screens/backupWalletSteps/verifySeed.tsx:102]: handled in latest commit

File: src/app/routes/index.tsx [hunk 2]
```diff
@@ -36,10 +36,6 @@ import RestoreOrdinals from '@screens/restoreFunds/restoreOrdinals';
 import ImportLedger from '@screens/ledger/importLedgerAccount';
 import VerifyLedger from '@screens/ledger/verifyLedgerAccountAddress';
 import ConfirmLedgerTransaction from '@screens/ledger/confirmLedgerTransaction';
-import LedgerSendStxScreen from '@screens/ledger/ledgerSendStx';
```
[the remark this answers is not in the document]
Comment (teebszet) [reply src/app/routes/index.tsx:39]: it should be pretty easy to add it back right? the webpack won't even build if the import is missing

**Pull request merged**